.travis.yml

```diff
@@ -27,26 +27,22 @@
 ---
 cache: pip
 dist: xenial
-addons:
-  apt:
-    update: true  # instruct travis-ci to always run apt-get before each build
 
 language: python
-
-stages:
-- documentation
+python:
+  - "3.7"
```

Comment:
Funny that other PRs failed due to missing sphinx module (https://github.com/cylc/cylc-doc/issues/32) but not this one.

Maybe py 3.7 in travis includes the sphinx module?

Reply:
Probably because I've added `pip install -e .` to the `.travis.yml`?

Reply:
Ah, that makes sense!